### ISSUE TITLE
Docs

### ISSUE THREAD

**Issue by ptmt** [ptmt/react-native-desktop#13]
- [ ] Finish React Browser https://github.com/ptmt/react-browser to be able easily try examples. 
- [ ] Add "Getting Started" introduction;
- [ ] Fix CLI Template (#12);
- [ ] Update `website` task to publish API docs;

**Comment by hkasera** [comment]
@ptmt The "[Getting Started](https://github.com/ptmt/react-native-desktop#getting-started)" instructions were quite confusing to me specially the last instruction.

`open osx/MyProject.xcodeproj`

I think it would be great if we can be explicit that **osx** is the folder in which the code is residing. Also how did we generate the **MyProject.xcodeproj** file. Unless I open the folder **MyProject** in xcode, I don't see this file. 

Let me know if I am missing something here! I would be happy to send a pull request to add these if it is a welcome contribution.

Thanks,
Harshita

**Comment by ptmt** [comment]
@hkasera thanks for pointing this out, I've added the additional command. Sorry, I somehow missed you comment here. It's certainly not a perfect, requires a lot of hidden knowledge and implicit actions (like you should install XCode). Need to fix that with the docs, which should be published to gh-pages eventually.

**Comment by hkasera** [comment]
@ptmt Sure, thanks :+1:

**Comment by tommoor** [comment]
Hey @ptmt - looks like this is a great project, thanks for all the hard work! It looks like in the latest release you implemented hot reloading? Does this change the getting started above?

**Comment by andrerfneves** [comment]
@ptmt do you want help with this getting started documentation? I see the tag on the issue.

**Comment by ptmt** [comment]
@andrerfneves yeah, it would be nice, though, I'm not sure how to solve the docs problem best. It's almost the same experience as React Native for iOS. I guess adding some pages here https://github.com/ptmt/react-native-macos/tree/master/docs will be the best. Then we can push it as a github pages

**Comment by andrerfneves** [comment]
@ptmt I could strip out some of those iOS/Android specific md files and do a quick revise of the other ones, and we can deploy to GH pages and then iterate on a branch until Docs are up-to-date/correct.

**Comment by andrerfneves** [comment]
@ptmt Do we want to completely erase .md files that don't pertain to macOS (or better, that pertain specifically to iOS / Android)?
